packages/notifi-frontend-client/lib/storage/InMemoryStorageDriver.ts

```diff
@@ -32,6 +32,7 @@ export const createInMemoryStorageDriver = (
       keyPrefix += `:${config.walletPublicKey}`;
       break;
     }
+    case 'ACALA':
     case 'APTOS': {
       keyPrefix += `:${config.accountAddress}:${config.authenticationKey}`;
       break;
```

Comment:
In react-hooks, the implementation [does not have chain specific storage prefix](https://github.com/notifi-network/notifi-sdk-ts/blob/main/packages/notifi-react-hooks/lib/utils/storage.ts#L32). All is set as `${jwtPrefix}:${dappAddress}:${walletPublicKey}`. But in some chains, the publickey is not constant. 
I am wondering if it will be good idea to use `${jwtPrefix}:${dappAddress}:${accountAddress}` for those chains that have accountAddress.

```suggestion
    case 'ACALA':
    case 'APTOS': {
      keyPrefix += `:${config.accountAddress}`;
      break;
    }
```